executor/write.go

```diff
@@ -833,7 +833,7 @@ func (e *InsertValues) getRow(cols []*table.Column, list []expression.Expression
 func (e *InsertValues) getRowsSelect(cols []*table.Column) ([][]types.Datum, error) {
 	// process `insert|replace into ... select ... from ...`
 	if e.SelectExec.Schema().Len() != len(cols) {
-		return nil, errors.Errorf("Column count %d doesn't match value count %d", len(cols), e.SelectExec.Schema().Len())
+		return nil, ErrWrongValueCountOnRow.GenByArgs(1)
```

Comment:
Because select list can be treated as the first row of **VALUES**. So both of statements:
```sql
insert into T VALUES (1, 1), (2, 2);
insert into T SELECT 1, 1;
```
will return error message:
```
ERROR 1136(21S01): Column count doesn't match value count at row 1
```